packages/apps/human-app/server/src/modules/oracle-discovery/spec/oracle-discovery.controller.spec.ts

```diff
@@ -45,28 +38,9 @@ describe('OracleDiscoveryController', () => {
     controller = module.get<OracleDiscoveryController>(
       OracleDiscoveryController,
     );
-    serviceMock = module.get<OracleDiscoveryService>(OracleDiscoveryService);
   });
 
   it('should be defined', () => {
     expect(controller).toBeDefined();
   });
-
-  // describe('oracle discovery', () => {
```

Comment:
Added back in https://github.com/humanprotocol/human-protocol/commit/e80d95e9e977c54e66cc49e0dda1585dd9905ea8

Reply:
can you uncomment? it might need some adjustments

Reply:
@portuu3 done in https://github.com/humanprotocol/human-protocol/pull/2588/commits/9fb3238e4b65157ddf35b7bf82dd8e65f9869769

Uncommenting these tests will make CI fail because `jobsDiscoveryFlag` is not enabled ([check this sample github actions run](https://github.com/humanprotocol/human-protocol/actions/runs/11218807112/job/31183387183?pr=2588)). Do you think we need to uncomment and fix it in this PR? If we want to enable them I would go in a separate one

Reply:
undo this commit then and I will create a new issue to do it in a different PR

Reply:
Done. Also rebased latest `develop`